app/controllers/albums.js

```diff
@@ -0,0 +1,19 @@
+const albumService = require('../services/albums');
+
+exports.getAlbums = async (req, res, next) => {
+  try {
+    const albumsRequest = await albumService.getAlbums();
+    res.send(albumsRequest);
+  } catch (error) {
+    next(error);
+  }
+};
+
+exports.getPictures = async (req, res, next) => {
+  try {
+    const picturesRequest = await albumService.getPictures(req);
```

Comment:
`pictures` instead of `picturesRequest` seems good, because you are getting the pictures